src/rgw/rgw_perf_counters.h

```diff
@@ -99,3 +108,20 @@ void tinc(const CountersContainer &counters, int idx, utime_t);
 void tinc(const CountersContainer &counters, int idx, ceph::timespan amt);
 
 } // namespace rgw::op_counters
+
+namespace rgw::persistent_topic_counters {
+
+class CountersContainer {
```

Comment:
@cbodley what do you think about another structure named `CountersContainer` even though it's in a different namespace?

Reply:
not a problem in general, that's what namespacing is for. the purpose of this `CountersContainer` is slightly different than the original one - it's just responsible for removing the instance from its collection, and doesn't hold multiple instances. but `CountersContainer` is still a reasonable name for it